referentiel_evaluation.md

```diff
@@ -310,9 +316,10 @@ Complément sur l'utilisation de données synthétiques et d'approches de _data
 <details>
 <summary>Ressources2.3 :</summary>
 
-- *[Measuring fairness](https://pair.withgoogle.com/explorables/measuring-fairness)* explorable, [PAIR](https://pair.withgoogle.com/)
-- _[AI Fairness 360](https://developer.ibm.com/technologies/artificial-intelligence/projects/ai-fairness-360/): an open source software toolkit that can help detect and remove bias in machine learning models_, IBM
-- _Fairness metrics_ : _[counterfactual fairness](https://papers.nips.cc/paper/6995-counterfactual-fairness)_, _[adversarial debiaising](https://arxiv.org/pdf/1801.07593.pdf)_
+- (Web article) *[Measuring fairness](https://pair.withgoogle.com/explorables/measuring-fairness)* explorable, [PAIR](https://pair.withgoogle.com/)
```

Comment:
This line need to be split into 2 lines in order to respect the rule "one resource by line"

Reply:
Ca serait bizarre de séparer le site de l'article, je préfère laisser cette ressource et sa source telle quelle